### PR TITLE
Proposed updates to System Console help texts

### PR DESCRIPTION
**Pull request by jasonblais**

### CODE REVIEW

File: plugin.json [hunk 1]
```diff
@@ -12,30 +12,30 @@
     },
     "settings_schema": {
         "header": "",
-        "footer": "Use this webhook URL to set up the StatusPage integration.\n\n`https://SITEURL/plugins/statuspage/webhook?service=<NAME_OF_THE_SERVICE>&token=<TOKEN>`",
+        "footer": "Use this webhook URL to set up the Statuspage integration.\n\n`https://SITEURL/plugins/statuspage/webhook?service=NAMEOFSERVICE>&token=TOKEN`",
         "settings": [{
-            "key": "Team",
+            "key": "Team:",
```
Comment (cpanato) [inline plugin.json]: The key should be like that. you might want to change the `display_name`

File: plugin.json [hunk 2]
```diff
@@ -12,30 +12,30 @@
...
             "display_name": "Team",
             "type": "text",
-            "help_text": "Select the team that this integration is attached to."
+            "help_text": "Team you want to send messages to. Use team URL, instead of a display name."
         },
         {
-            "key": "Channel",
+            "key": "Channel:",
```
Comment (cpanato) [inline plugin.json]: same here

File: plugin.json [hunk 3]
```diff
@@ -12,30 +12,30 @@
...
             "display_name": "Channel",
             "type": "text",
-            "help_text": "Name of the Channel you want to create/use to post the messages."
+            "help_text": "Channel you want to send messages to. Use channel URL such as 'town-square', instead of a display name. If you specify a channel that does not exist, this plugin creates a new channel with that name."
         },
         {
-            "key": "Username",
+            "key": "Username:",
```
Comment (cpanato) [inline plugin.json]: same here

File: plugin.json [hunk 4]
```diff
@@ -12,30 +12,30 @@
...
             "display_name": "User",
             "type": "username",
             "help_text": "Select the username that this integration is attached to."
         },
         {
-            "key": "Token",
+            "key": "Token:",
```
Comment (cpanato) [inline plugin.json]: and same here

**Comment by jasonblais** [comment]
Updated the display name instead of the key 👍

**Pull request merged**